valet

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 SOURCE="${BASH_SOURCE[0]}"
+SUDOCMDS="install start stop secure share"
```

Comment:
This was the part I mentioned in the original PR, I had to pull in @willvincent PR #206 to address issues I was having with the Brew methods wanting to run as sudo. I can't get the versions to switch without this, it wants me to run the command as sudo.